packages/dds/tree/src/test/feature-libraries/flex-map-tree/mapTreeNode.spec.ts

```diff
@@ -21,26 +16,24 @@ import { leaf as leafDomain } from "../../../domains/index.js";
 import { brand } from "../../../util/index.js";
 // eslint-disable-next-line import/no-internal-modules
 import { getOrCreateMapTreeNode } from "../../../feature-libraries/flex-map-tree/index.js";
+import { getFlexSchema, SchemaFactory } from "../../../simple-tree/index.js";
 
 describe("MapTreeNodes", () => {
 	// #region The schema used in this test suite
-	const schemaBuilder = new SchemaBuilderBase(FieldKinds.required, {
-		scope: "Test",
-		libraries: [leafDomain.library],
-	});
-	const mapSchema = schemaBuilder.map(
-		"Map",
-		FlexFieldSchema.create(FieldKinds.optional, [leafDomain.string]),
-	);
-	const arrayNodeSchema = schemaBuilder.object("ArrayNode", {
-		[EmptyKey]: FlexFieldSchema.create(FieldKinds.sequence, [leafDomain.string]),
-	});
 	const objectMapKey = "map" as FieldKey;
 	const objectFieldNodeKey = "fieldNode" as FieldKey;
-	const objectSchema = schemaBuilder.object("Object", {
-		[objectMapKey]: mapSchema,
-		[objectFieldNodeKey]: arrayNodeSchema,
+
+	const schemaBuilder = new SchemaFactory("Test");
```

Comment:
nit: schemaFactory (or more typically in our codebase, sf) is a better name

Reply:
Will rename this (and several others) in a future change